Common/AR/include/Android/ArCoreWrapper.h

```diff
@@ -146,7 +146,7 @@ class ArCoreWrapper
   ArCamera* m_arCamera = nullptr;
 
   // data returned from each frame
-  std::array<float, 8> m_transformedUvs;
+  std::array<float, 8> m_transformedUvs = {};
```

Comment:
Is this needed?

Reply:
This is a warning from clang-tidy, to default initialize the std::array.